htdocs/core/class/html.form.class.php

```diff
@@ -7895,6 +7895,22 @@ public function showrefnav($object, $paramid, $morehtml = '', $shownav = 1, $fie
 			$fieldref = 'ref';
 		}
 
+		// Preparing gender's display if there is one
+		if ($object->gender) {
```

Comment:
The showrefnav may be used for all object. And some object does not contains the property gender. So to avoid error with php8, you must also add a test here
`if (property_exists($object, 'gender') && $object->gender)`